src/OrbitGl/App.cpp

```diff
@@ -2365,7 +2365,7 @@ Future<ErrorMessageOr<CanceledOr<std::filesystem::path>>> OrbitApp::RetrieveModu
 
             if (previous_result.has_value()) return {previous_result.value()};
 
-            return RetrieveModuleViaDownload(module_id).Then(
+            return RetrieveModuleFromRemote(module_id).Then(
```

Comment:
Rename the future from `retrieve_via_download_future` to `retrieve_from_remote_future`.